package.json

```diff
@@ -8,7 +8,7 @@
     "debug": "^2.2.0",
     "dotenv": "^1.2.0",
     "express": "^4.13.3",
-    "express-ftwebservice": "git+http://git.svc.ft.com:8080/scm/ot/express-ftwebservice.git",
+    "express-ftwebservice": "^2.0.0",
```

Comment:
Yes, it is on npm and github now
https://www.npmjs.com/package/express-ftwebservice
https://github.com/Financial-Times/express-web-service